src/ts/greedy-nav/GreedyNav.test.ts

```diff
@@ -286,7 +287,7 @@ describe('Greedy Nav', () => {
     });
 
     it('toggles the menu open', () => {
-      const event = new dom.window.FocusEvent('focus');
+      const event = new dom.window.KeyboardEvent('keyup', { key: 'Tab' });
       nav.navDropdownToggle!.dispatchEvent(event);
```

Comment:
Does testing libraries `fireEvent` work here? I feel like I wrote this in a weird way - not sure why.

Reply:
yup - all changed